keyboards/crkbd/keymaps/rpbaptist/rules.mk

```
@@ -0,0 +1,34 @@
+# This enables Link Time Optimization. This can save a good chunk of space (several KB for me), but the macro and function ... functions cause it to error out.
+EXTRAFLAGS        += -flto
```

This would also work: 
```suggestion
LINK_TIME_OPTIMIZATION_ENABLE = yes
```